docs/iot/guides/zigbee.md

```diff
@@ -0,0 +1,218 @@
+# Zigbee
```

Comment:
Question. This page is same with the page under native guide.
I've heard that it is requested gather Zigbee under IoT, but really two same pages are needed at two place?

If you are to keep this page under iot/guides/,
please update the link on line 22.
I think "../../application/tizen-studio/common-tools/emulator.md" will work.

Reply:
Regarding redundant pages, I checked with @saerome and I have removed the guide present in native section.
Zigbee should only be added to IOT section.

Also I have updated the emulator link.